ksqldb-functional-tests/src/test/resources/rest-query-validation-tests/scalable-push-queries.json

```diff
@@ -0,0 +1,773 @@
+{
+  "comments": [
+    "Tests covering Scalable Push queries"
+  ],
+  "tests": [
+    {
+      "name": "only key column",
+      "statements": [
+        "CREATE STREAM I (K STRING KEY, ID INT) WITH (kafka_topic='test_topic', value_format='JSON', partitions=1);",
+        "CREATE STREAM INPUT as SELECT * FROM I;",
+        "SELECT K FROM INPUT EMIT CHANGES LIMIT 2;"
+      ],
+      "inputs": [
+        {"topic": "test_topic", "timestamp": 12345, "key": "11", "value": {"id": 100}},
+        {"topic": "test_topic", "timestamp": 12365, "key": "12", "value": {"id": 101}}
+      ],
+      "inputConditions": {
+        "waitForQueryHeader": true
+      },
+      "properties": {
+        "auto.offset.reset" : "latest"
+      },
+      "responses": [
```

Comment:
is there any indication that it indeed issued a scalable push query and not just a regular old push query? we should test for that as well in case of regression

Reply:
I was debating how to do that.  The queryid is prefixed by SCALABLE_PUSH_.  That might be a decent way to check this.  Alternatively, there could be a force SPQ flag that would error the query if it wasn't an SQP, but that seems like not that right approach since they're meant to happen transparently.

Reply:
for now I think using the `queryID` is OK, but we should think about a way to expose to the user whether what they issued was scalable or not (one option could be some kind of `EXPLAIN` for push queries) then our framework can always issue the explain before issuing the query and fail if it isn't scalable. 

Can you create tickets to track this (one for exposing what type of push query it was to the user and one for updating RQTTs to leverage that)?

Reply:
I added the ability to check the queryId prefix and am able to verify it's a scalable push query.